tests/component/test_task_resource_warnings.py

```diff
@@ -0,0 +1,45 @@
+import warnings
+import nidaqmx
+
+import pytest
+
+from nidaqmx.errors import DaqResourceWarning
+
+
+@pytest.mark.library_only
+def test__unclosed_task__leak_task__resource_warning_raised(task):
```

Comment:
In test function names, separate given/when/then with 3 underscores.

Reply:
Added underscore in function names.